keps/sig-node/2400-node-swap/README.md

```diff
@@ -622,12 +650,24 @@ checking if there are objects with field X set) may be a last resort. Avoid
 logs or events for this purpose.
 -->
 
+KubeletConfiguration has set `failOnSwap: false`.
+
+The prometheus `node_exporter` will also export stats on swap memory
+utilization.
+
 ###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
 
 <!--
 Pick one more of these and delete the rest.
 -->
 
+TBD. We will determine a set of metrics as a requirement for beta graduation.
```

Comment:
I think it would be good to add `Swap` to the kubelet `MemoryStats` API so we can track usage of swap by kernel for both host and end-user workloads.